_config.yml

```diff
@@ -80,7 +80,7 @@ react_mini_router:
 translations:
   es: v2.2.4
   fr: v2.5.0
-  ja: v3.13.1
+  ja: v3.13.2
```

Comment:
Thank you for your PR !!
But, plz don't change here. Because the version of riot itself is `3.13.1` 🙇

Reply:
Ooops!
I will send again pull request.